src/auth_db/mod.rs

```diff
@@ -162,6 +247,28 @@ impl Db for DbClient {
             status => Err(DbError::new(format!("auth db response: {}", status))),
         }
     }
+
+    fn create_bounce(
+        &self,
+        address: &str,
+        bounce_type: BounceType,
+        bounce_subtype: BounceSubtype,
+    ) -> Result<(), DbError> {
+        let response = self
+            .request_client
+            .post(self.urls.create_bounce())
+            .json(&BounceRecord {
+                address: address.to_string(),
+                bounce_type,
+                bounce_subtype,
+                created_at: 0,
+            })
+            .send()?;
```

Comment:
This is also synchronous of course, but I'm not targeting it as part of the futures stuff I'm doing right now. We can come back to it later but also I'm conscious that long-term the db stuff is earmarked for big changes anyway, so perhaps it's not worth it.